### PR TITLE
Update 03-guides.md

### PR DESCRIPTION
**Pull request by tkurian123**
Included the AMEX point of sale details in FAQ guide..

### CODE REVIEW

File: apis/01-Availability, Rates, Restrictions, Booking Notification, Retrieval and Confirmation/02-Booking-Notification-API/03-guides.md [hunk 1]
```diff
@@ -214,7 +226,7 @@ Hotel Collect bookings sourced from Hotels.com points of sales
 	</POS>
 ```
 
-Hotel Collect bookings sourced from Travelocity points of sales: 
+Hotel Collect bookings sourced from Travelocity point of sales:
```
Comment (EXPEylazzari) [inline apis/01-Availability, Rates, Restrictions, Booking Notification, Retrieval and Confirmation/02-Booking-Notification-API/03-guides.md]: Is the correct plural form "point of sales"? In that case, how about changing it everywhere, at least on this page? It seems everywhere else it's written as "point**s** of sales".

Comment (tkurian123) [inline apis/01-Availability, Rates, Restrictions, Booking Notification, Retrieval and Confirmation/02-Booking-Notification-API/03-guides.md]: You are right..It should be 'Points of Sale'. Correcting..

**Comment by jhitzke** [comment]
Is this PR still needed or is it able to be closed?

**Comment by tkurian123** [comment]
Hi Joe,

This may be closed ; thanks
-Thomas

**Pull request closed without merging**